src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/api/ReportTests.kt

```diff
@@ -33,6 +35,27 @@ class ReportTests : IntegrationTest()
         assertSuccessfulWithResponseText(response)
         assertJsonContentType(response)
         JSONValidator.validateAgainstSchema(response.text, "Run")
+
+        val key = JSONValidator.getData(response.text)["key"].asText()
+        apiRequestHelper.delete("/reports/$key/kill/", userEmail = fakeGlobalReportReviewer())
+    }
+
+    @Test
+    fun `kills report`()
+    {
+        val runResponse = apiRequestHelper.post("/reports/minimal/run/", mapOf(),
+                userEmail = fakeGlobalReportReviewer())
+        assertSuccessfulWithResponseText(runResponse)
+        val key = JSONValidator.getData(runResponse.text)["key"].asText()
+
+        //Until https://mrc-ide.myjetbrains.com/youtrack/issue/VIMC-3849 is fixed, the orderly server endpoint works but
```

Comment:
..again we can modify this test to expect a 200 once this other orderly server ticket is done

Reply:
lol, ok